src/process/windows/SDL_windowsprocess.c

```diff
@@ -489,8 +533,10 @@ void SDL_SYS_DestroyProcess(SDL_Process *process)
     if (io) {
         SDL_CloseIO(io);
     }
-    CloseHandle(data->process_information.hThread);
-    CloseHandle(data->process_information.hProcess);
+    if (data) {
+        CloseHandle(data->process_information.hThread);
```

Comment:
We should initialize with INVALID_HANDLE_VALUE and check for that here. CloseHandle() will generate an exception if passed an invalid handle.